docs/SchemaMigrationTool.md

```diff
@@ -23,30 +23,26 @@ Note - The tool can't downgrade a schema version.
 
 - #### How to install the tool
 
-    - ##### If available as .NET Core global tool 
+    - ##### If available as .NET Core global tool (not available yet)
```

Comment:
Why do we document this yet if it isn't available?

Reply:
Makes sense! Removed for now and I will add back when it will be available as global tool.